### PR TITLE
Add support for platformId in success event payload

### PR DESCRIPTION
**Pull request by rawbee**
## Description of the change

Add support for platformId in success event payload

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (TrekSoft) [inline .vscode/launch.json:1]: Just want to double check this was intentional adding this file?

**Review by TrekSoft — APPROVED** [review]
possum2_1

File: .vscode/launch.json (hunk 1 above)
Comment (rawbee) [reply .vscode/launch.json:1]: It is! Sets up debugging in VSCode.

Comment (TrekSoft) [reply .vscode/launch.json:1]: Beautiful and loveley!

**Pull request merged**